Set level to info

prateekj117 opened the pull request:
<!--
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->
<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes #5722 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The unit tests pass locally with my changes <!-- use `nosetests tests/all` to run all the tests -->
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

uds5501 commented:
@prateekj117 setting the level to `INFO` also prints the `Saved to db` and `added to session` type logs. Is it desirable?

prateekj117 commented:
@uds5501 It does help in understanding which end points it is targeting, plus success messages, so it may help in developing, your views on this?

uds5501 commented:
@prateekj117 I am pro this change, although you might want to address the multiple migration error

prateekj117 commented:
@uds5501 On it.

prateekj117 commented:
@iamareebjamal Running python manage.py db heads gives:
Raven is not configured (logging is disabled). Please see the documentation for more information.
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Added job "sync_events_elasticsearch" to job store "default"
Added job "cron_rebuild_events_elasticsearch" to job store "default"
Added job "send_after_event_mail" to job store "default"
Added job "send_event_fee_notification" to job store "default"
Added job "send_event_fee_notification_followup" to job store "default"
Scheduler started
6e5c574cbfb8 (head)

iamareebjamal commented:
2 things, INFO messages of libraries should not be printed. Only our INFO messages should be printed.

Secondly, check the return code of the migration head command. It should be 0. If it is non-zero, multiple migration head command will be executed

iamareebjamal commented on app/__init__.py:
```diff
@@ -100,19 +100,18 @@ def create_app():
             'default': {
                 'handlers': ['console'],
-                'level': logging.INFO
+                'level': logging.WARN
```
Are requests being logged now?

And only INFO from our application gets printed?

iamareebjamal replied:
I think opposite will happen. INFO from libraries will get print and from our app won't

prateekj117 replied:
@iamareebjamal Yes, requests are being logged.

prateekj117 commented:
@iamareebjamal Regarding this, I think we should keep the grep change, and discard every other one. That would solve I suppose #5650.
